src/ui/canvas/gdi/Bitmap.cpp

```diff
@@ -26,7 +26,9 @@ Copyright_License {
 
 #include "Screen/Debug.hpp"
 #include "system/Path.hpp"
+#include "LogFile.hpp"
```

Comment:
Why?

Reply:
Because LogFormat lines, - if they not necessary, this can be removed!